features/routing/route.feature

```diff
@@ -200,41 +200,15 @@ Feature: Testing route
       | f | service_secure.yaml |
     Then the step should succeed
 
-    Given I have a pod-for-ping in the project
-    And CA trust is added to the pod-for-ping
-    Given I obtain test data file "routing/reencrypt/route_reencrypt-reen.example.com.crt"
-    Given I obtain test data file "routing/reencrypt/route_reencrypt-reen.example.com.key"
-    Given I obtain test data file "routing/reencrypt/route_reencrypt.ca"
+    # check reencrypt route
     Given I obtain test data file "routing/reencrypt/route_reencrypt_dest.ca"
     When I run the :create_route_reencrypt client command with:
-      | name       | route-recrypt                                                                                 |
-      | hostname   | <%= rand_str(5, :dns) %>-reen.example.com                                                     |
-      | service    | service-secure                                                                                |
-      | cert       | route_reencrypt-reen.example.com.crt |
-      | key        | route_reencrypt-reen.example.com.key |
-      | cacert     | route_reencrypt.ca                   |
-      | destcacert | route_reencrypt_dest.ca              |
-      | path       | /test                                                                                         |
+      | name       | route-recrypt           |
+      | service    | service-secure          |
+      | destcacert | route_reencrypt_dest.ca |
```

Comment:
```
| path       | /test            |
```
should be kept since this case is testing the route with a path.

Reply:
Thanks for the review. I have added back the path, and modified the two checked points.